com.unity.multiplayer.mlapi/Runtime/LagCompensation/LagCompensationManager.cs

```diff
@@ -40,9 +40,12 @@ public static void Simulate(float secondsAgo, Action action)
         /// <param name="action">The action to invoke when time is turned back</param>
         public static void Simulate(float secondsAgo, IList<TrackedObject> simulatedObjects, Action action)
         {
-            if (!NetworkingManager.Singleton.IsServer)
+            if( simulatedObjects.Count > 0 )
             {
-                throw new NotServerException("Only the server can perform lag compensation");
+                if( !simulatedObjects[0].NetObject.NetManager.IsServer )
```

Comment:
Oh boy, I guess you **_could_** however unlikely, have a collection of simulatedObjects were some are from one NM that is set up as a server and some are as a client.  I haven't worked with LagCompensationManager though.

Is this telling us that this should be an instance method?  There should be an LCM per NM?